client/screens/sidePanel/sidePanel.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {AddressBook, CategoryFilter} from '../../components'
+import {Link} from 'react-router-dom'
+
+class ScreensidePanel extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      addressBook: true
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick() {
+    this.setState({addressBook: !this.state.addressBook})
+  }
+
+  render() {
+    return (
+      <div id="side-panel">
+        <div>
+          <Link to="/addressBook" onClick={this.handleClick}>
+            Address Book
+          </Link>
+          <Link to="/categoryFilter" onClick={this.handleClick}>
+            Category Filter
+          </Link>
+        </div>
+        {this.state.addressBook ? (
+          <div>
+            <AddressBook />
+          </div>
+        ) : (
+          <div>
+            <CategoryFilter />
+          </div>
+        )}
+      </div>
+    )
+  }
+}
+
+export default ScreensidePanel
```

Comment:
Since we're using Pascal Case, can we call this component `ScreensSidePanel`?

Reply:
Fixed